src/main/java/io/sudhakar/student/dto/Employee.java

```diff
@@ -0,0 +1,20 @@
+package io.sudhakar.student.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Employee {
```

Comment:
have you tested this class there's no data or getter setter

Reply:
added getter setter